README.md

```diff
@@ -5,33 +5,66 @@ both web and API testing, wraps and extends Selenium calls to more appropriately
 and supports testing over multiple browsers locally, or in the cloud (Selenium Grid or SauceLabs) in 
 parallel. It can be a great starting point for building or improving test automation in your organization.
 
-## Installation
-### Building the jar
-If you want to compile the jar from the source code, use maven. Maven can be used to run unit tests, run
-integration tests, build javadocs, and build the executable jar. To simply execute the unit tests, run the
-below command
-```
-mvn clean test
-```
-To also build the jars, run the below commands
-```
-mvn clean package
-```
-To run the integration tests, use the verify goal. The integration tests currently point at a private server
-hosting the file found in this base directory called `index.html`. In order to properly execute these tests,
-host this file, and set the testSite to point to the hosted file's location. This can be done dynamically through
-the command line, as outlined below in the Application URL section.
+## Test Examples
+### Getting Started
+One of Selenified's goals is to be a framework that is easy to drop in to an existing project. You can 
+easily have Selenified running within minutes using only a Maven POM, Java test class and a TestNG XML Suite. 
+Have a look at this example test class to get an idea of what you'll actually be adding into your codebase.
 
-Some of the integration tests require a physical browser to run, and so they can be run two different ways, the 
-entire set with a browser, or a subset using HtmlUnit
-```
-mvn clean verify -Dbrowser=Firefox
-mvn clean verify -Dfailsafe.groups=virtual
+```java
+    public class SampleTests extends TestBase {
+
+        @BeforeClass(alwaysRun = true)

```

Comment:
We should remove this from our simple example, as it's not truly needed. We should add it in same as talking about overridding other methods